src/Common/Storage/Commands.Storage/Blob/Cmdlet/SetAzureStorageContainerStoredAccessPolicy.cs

```diff
@@ -25,7 +25,7 @@ namespace Microsoft.WindowsAzure.Commands.Storage.Blob.Cmdlet
     /// <summary>
     /// create a new azure container
     /// </summary>
-    [Cmdlet(VerbsCommon.Set, StorageNouns.ContainerStoredAccessPolicy), OutputType(typeof(String))]
+    [Cmdlet(VerbsCommon.Set, StorageNouns.ContainerStoredAccessPolicy, SupportsShouldProcess=true, ConfirmImpact = ConfirmImpact.Medium), OutputType(typeof(String))]
     public class SetAzureStorageContainerStoredAccessPolicyCommand : StorageCloudBlobCmdletBase
```

Comment:
@EmmaZhu Every cmdlet that makes permanent chenges should implement ShouldProcess